src/router/index.js

```diff
@@ -9,17 +10,22 @@ const routes = [
   },
   {
     path: "/about",
-    name: "About",
+    // name: 'About',
     // route level code-splitting
     // this generates a separate chunk (about.[hash].js) for this route
     // which is lazy-loaded when the route is visited.
     component: () =>
       import(/* webpackChunkName: "about" */ "../views/About.vue"),
   },
+  {
+    path: "/game-details",
+    name: "GameDetails",
+    component: GameDetails,
+  },
 ];
 
 const router = createRouter({
-  history: createWebHashHistory(),
+  history: createWebHistory(),
```

Comment:
I've read somewhere which said this may cause some routing issue when the app is deployed to s3. Not sure if that's the case though until we test it out.

Reply:
I see, because I tried createWebHashHistory and the url becomes weird that I can't change page by changing the url 🧐🧐